tests/nested/core20/basic/task.yaml

```diff
@@ -30,4 +30,7 @@ execute: |
     nested_exec "sudo snap remove test-snapd-sh" || true
 
     echo "Ensure 'snap list' works and test-snapd-sh snap is removed"
-    ! nested_exec "snap list test-snapd-sh"
```

Comment:
Could you replace this with `nested_exec "! snap list test-snapd-sh"` - this is short and has the right semantics.

Reply:
Updated and force pushed (so that I can cherrypick into 2.47)